recipes/date/all/conanfile.py

```diff
@@ -40,6 +41,11 @@ def _configure_cmake(self):
         cmake.definitions["ENABLE_DATE_TESTING"] = False
         cmake.definitions["USE_SYSTEM_TZ_DB"] = self.options.use_system_tz_db
         cmake.definitions["USE_TZ_DB_IN_DOT"] = self.options.use_tz_db_in_dot
+        cmake.definitions["BUILD_TZ_LIB"] = not self.options.header_only
+        # workaround for clang 5 not having string_view
+        if self.version == "3.0.0" and self.settings.compiler == "clang" \
```

Comment:
This should probably be `>= 3.0.0`?

Reply:
The author [decided](https://github.com/HowardHinnant/date/releases/tag/v3.0.0) this is a final version. But who knows?